cli.go

```diff
@@ -106,8 +107,13 @@ func (cli *CLI) Run(args []string) int {
 
 	// If disabled test mode, generates real detector
 	if !cli.testMode {
-		templates, state, tfvars := cli.loader.Dump()
-		cli.detector, err = detector.NewDetector(templates, state, tfvars, c)
+		templates, files, state, tfvars := cli.loader.Dump()
+		schema, err := schema.Make(files)
+		if err != nil {
+			fmt.Fprintln(cli.errStream, fmt.Errorf("ERROR: Parse error: %s", err))
+			return ExitCodeError
+		}
+		cli.detector, err = detector.NewDetector(templates, schema, state, tfvars, c)
```

Comment:
__[gometalinter]__
* [ineffassign] ineffectual assignment to err
* [staticcheck] this value of err is never used (SA4006)

<!-- Comment by [SideCI](https://sideci.com) -->